.github/workflows/check.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version: "lts/*"
-          cache: 'npm'
+          cache: "npm"
```

Comment:
did the auto formatter change?

Reply:
This file was added back in August, I think I didn't have the project vscode config set to format on save back then